shared_model/validators/container_validator.hpp

```diff
@@ -34,16 +34,23 @@ namespace shared_model {
         }
       }
 
-     public:
-      explicit ContainerValidator(
-          const FieldValidator &field_validator = FieldValidator(),
-          const TransactionsCollectionValidator
-              &transactions_collection_validator =
-                  TransactionsCollectionValidator())
+      explicit ContainerValidator(const FieldValidator &field_validator,
```

Comment:
please use
```suggestion
      explicit ContainerValidator(FieldValidator field_validator,
```
& `move()` since it is acquired